consensus/istanbul/backend/engine_test.go

```diff
@@ -759,37 +764,48 @@ func makeSnapshotTestConfigItems(stakingInterval, proposerInterval uint64) []int
 	}
 }
 
+func makeMockStakingManager(t *testing.T, amounts []uint64, blockNum uint64) (*gomock.Controller, *mock.MockStakingModule) {
+	si := makeTestStakingInfo(amounts, blockNum)
+
+	mockCtrl := gomock.NewController(t)
+	mStaking := mock.NewMockStakingModule(mockCtrl)
+	mStaking.EXPECT().GetStakingInfo(gomock.Any()).Return(si, nil).AnyTimes()
+	return mockCtrl, mStaking
```

Comment:
https://pkg.go.dev/github.com/golang/mock/gomock#Controller.Finish
It seems `Finish()` is not required. I think we can remove `mockCtrl` if we're only returning it for `Finish()`.

Reply:
Sounds nice. Let me delete it later though.